app.js

```diff
@@ -14,7 +15,7 @@ class App {
 
         // Map conference ID to state about that conference
         // Conference state contains, at least:
-        // dumpId: (String) the dump ID for this conference
+        // clientId: (String) the dump ID for this conference
```

Comment:
I think `dumpId` actually works better here as it makes sense for both clients and jvb, but I assume it's too late to keep this as `dumpId`.

Reply:
It's not too late, we can change the naming conventions as we see fit, until we do a prod release at least.
We can switch back to dumpId. These naming conventions are basically legacy remains, and to be honest a lot of them are confusing so we might as well decide on some better naming conventions. How does statsSessionId sound?